.changeset/tough-dryers-greet.md

```diff
@@ -0,0 +1,11 @@
+---
```

Comment:
@lmagalhaes this a major backwards incompat change?

Reply:
The major release was from version 6 to 7, which introduces the AWS SDK upgrade, and this is merely a bug fix for the major version.
As all other systems still using `Steveo@6.*` there should be no impact.